TAScheduler/UnitTests/tests.py

```diff
@@ -11,38 +13,165 @@
 # Feel free to make suggestions on discord (add/remove/edit methods)!.
 ### Rememeber: These methods were made before any coding (I was guessing) so it's likely they should be changed.
 class TestUserLogin(TestCase):  # Alec
-    pass
-
-
-class TestUserGetID(TestCase):  # Alec
-    pass
+    def setUp(self):
+        self.user = User.objects.create(
```

Comment:
We need to use the classes from views_models.py. in that file user is an interface so you need to make an afmin/instructor/ta to test login.